sdk/src/main/java/com/sygic/travel/sdk/db/dao/PlaceDetailDao.kt

```diff
@@ -0,0 +1,28 @@
+package com.sygic.travel.sdk.db.dao
+
+import android.arch.persistence.room.Dao
+import android.arch.persistence.room.Delete
+import android.arch.persistence.room.Insert
+import android.arch.persistence.room.Query
+import com.sygic.travel.sdk.api.model.place.ApiPlaceDetail
+
+@Dao
+internal interface PlaceDetailDao {
+    @Query("SELECT * FROM place")
+    fun getAll(): List<ApiPlaceDetail>
+
+    @Query("SELECT * FROM place WHERE id IN (:arg0)")
+    fun loadAllByIds(ids: List<Int>): List<ApiPlaceDetail>
+
+    @Query("SELECT * FROM place WHERE id=:arg0")
```

Comment:
coding style id = :arg0
also, isn't possible to name args after the function arg's name?